ci.hocon

```diff
@@ -181,13 +181,39 @@ graal-vm-snapshot: {
   }
 }
 
+jruby-benchmark: {
+  setup: ${common-base.prelude}
+
+  downloads: {
+    JRUBY_HOME: {name: jruby, version: "9.1.12.0"}
```

Comment:
Can we use 9.1.7.0 for now? That's what we forked from.

Reply:
We could, if you feel strongly about it. I prefer 9.1.12.0 because it's the last JRuby release targeting the same MRI level as us (i.e., 2.3.3). And it's not 9 months out of date.

We could also test against both easily enough -- the jobs are only run weekly anyway.

Reply:
Can we just use 9.1.7.0 for now, and we can always update later?